src/opcode.cc

```diff
@@ -62,7 +59,9 @@ Opcode Opcode::FromCode(uint8_t prefix, uint32_t code) {
 }
 
 Opcode::Info Opcode::GetInfo() const {
-  return enum_ < Invalid ? infos_[enum_] : invalid_info_;
+  if (enum_ >= Invalid)
```

Comment:
I think we need to do something more clever in `Opcode::FromCode` above instead of returning `Opcode(Invalid)`. Maybe we can use a negative value for `enum_` to mean an invalid opcode with code `-enum_`.